en-US/dictionary.txt

```diff
@@ -1,41 +1,39 @@
-Aboleth
-Aboleths
++s
++'s
```

Comment:
This syntax means, that any word in the dictionary that ends in an asterisk may have one of these suffixes.
Any word in the dictionary that ends in a plus sign is _only_ valid with one of the suffixes we define. (We don't have any of those and I don't think we need any.)